conf/ssl4eo-l.yaml

```diff
@@ -0,0 +1,36 @@
+module:
+  _target_: torchgeo.trainers.MoCoTask
+  model: resnet50
```

Comment:
Ideally we'll want pre-trained models for ResNet-18, ResNet-50, and ViT-S. Would probably prioritize the last 2. If we don't have any ViTs I think the reviewers will laugh at us.

Reply:
I think we should focus on ResNet50 and ViT-S first. The only other SSL paper I've seen recently that trains ResNet18 is SeCo.